docs/api.rst

```diff
@@ -454,24 +467,193 @@ in these models see :ref:`sec_api_mutation_matrix_models_details`.
+frequencies. If this mutation model is used with a ``mutation_rate`` of
+:math:`\mu`, then the mutation rate from the i-th to the j-th allele is
+equal to :math:`Q_{ij}`, where
+
+:math:`\mathbf{Q} =
+\frac{\mu}{M} \times
+\begin{bmatrix}
+\cdot        &  \pi_C         & \kappa \pi_G & \pi_T           \\
+\pi_A        &  \cdot         & \pi_G        & \kappa \pi_T    \\
+\kappa \pi_A &  \pi_C         & \cdot        & \pi_T           \\
+\pi_A        &  \kappa \pi_C  & \pi_G        & \cdot
+\end{bmatrix}`
+
+Here :math:`M` is an overall scaling factor on the mutation rate that can
+be computed as follows: let :math:`q_{i,j} = \pi_j` if i<->j is a transversion,
+and :math:`q_{ij} = \kappa \pi_j` otherwise. Set :math:`q_{ii} = 0`, and
+then let :math:`M` be the largest row sum of :math:`q`, i.e.,
+:math:`M = \max_i \left( \sum_{j \neq i} q_{ij} \right)`.
+Then, this implementation as a :class:`.MatrixMutationModel` has the transition
+matrix :math:`P_{ij} = q_{ij} / M`, and :math:`P_{ii}` chosen so rows sum to one.
+
+Note also that :math:`\kappa` is the ratio of *individual* mutation rates,
+not the ratio of *total* transition to transversion rates, which would
+be :math:`\kappa/2`, as is used in some parameterizations.
+
+Citation: *Hasegawa M, Kishino H, Yano T (1985).  "Dating of the human-ape
+splitting by a molecular clock of mitochondrial DNA". Journal of Molecular
+Evolution. 22 (2): 160–74.*
+
+Note that this implementation has the root distribution as a separate
+parameter, although it defaults to the equilibrium distribution. If you
+set the root distribution to be different than the equilbrium distribution,
+then you have a nonequilibrium model, and you should make sure that's what
+you want.
+
 .. autoclass:: msprime.F84MutationModel()
 
+The Felsenstein 1984 mutation model of nucleotide mutation allows an arbitrary
+equilibrium nucleotide distribution, and different transition/transversion
+rates (although in a different way to HKY).
+For general notes on this and other MatrixMutationModels, see
+:ref:`sec_api_mutation_matrix_theory`.
+
+This model is parameterized by :math:`\kappa` (``kappa``), the ratio of
+transition to transversion mutation rates, and :math:`\pi`
+(``equilibrium_frequencies``), the vector of equilibrium nucleotide
+frequencies. If this mutation model is used with a ``mutation_rate`` of
+:math:`\mu`, then the mutation rate from the i-th to the j-th allele is
+equal to :math:`Q_{ij}`, where
+
+:math:`\mathbf{Q} =
+\frac{\mu}{M} \times
+\begin{bmatrix}
+\cdot        &  \pi_C         & \left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_G & \pi_T           \\
+\pi_A        &  \cdot         & \pi_G        & \left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_T    \\
+\left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_A &  \pi_C         & \cdot        & \pi_T           \\
+\pi_A        &  \left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_C  & \pi_G        & \cdot
+\end{bmatrix}`
```

Comment:
The equations here are not the same as in the code, I _think_ the code has this:
```suggestion
\begin{bmatrix}
\cdot        &  \pi_C         & \left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_G & \pi_T           \\
\pi_A        &  \cdot         & \pi_G        & \left(1 + \frac{\kappa-1}{\pi_C + \pi_T}\right) \pi_T    \\
\left(1 + \frac{\kappa-1}{\pi_A + \pi_G}\right) \pi_A &  \pi_C         & \cdot        & \pi_T           \\
\pi_A        &  \left(1 + \frac{\kappa-1}{\pi_C + \pi_T}\right) \pi_C  & \pi_G        & \cdot
\end{bmatrix}`
```

Reply:
whoops! good catch.